src/gui/AstroCalcDialog.cpp

```diff
@@ -5043,18 +5043,17 @@ void AstroCalcDialog::saveSolarEclipseMap()
 	// Find exact time of minimum distance between axis of lunar shadow cone to the center of Earth
 	const double JDMid = getJDofMinimumDistance(eclipseJD);
 
-	int year, month, day;
-	StelUtils::getDateFromJulianDay(JDMid, &year, &month, &day);
-
 	core->setUseTopocentricCoordinates(savedTopocentric);
 	core->update(0);
 
 	// Use year-month-day in the file name
-	const auto eclipseDateStr = QString("-%1-%2-%3").arg(year).arg(month,2,10,QChar('0')).arg(day,2,10,QChar('0'));
+	const auto eclipseDateStr = localeMgr->getPrintableDateLocal(JDMid);
+	const auto fileBaseName = qc_("Solar eclipse %1",
```

Comment:
The context data is very long. Maybe just comment for translators and short context (or without context)

Reply:
How would you formulate this in a shorter way, keeping it unambiguous?

Reply:
OK, I've dropped this, since the simple concatenation is already used in the caption inside KML.